readthedocs/proxito/views/mixins.py

```diff
@@ -68,6 +71,32 @@ def _serve_docs(
             download=download,
         )
 
+    def _trak_pageview(self, project, path, request):
+        """Create an audit log of the page view if audit is enabled."""
+        # Remove any query args (like the token access from AWS).
+        path_only = urlparse(path).path
+        track_file = any(
+            path_only.endswith(ext)
+            for ext in ['.html', '.pdf', '.epub', '.zip']
+        )
+
+        if track_file and self._is_audit_enabled(project):
+            AuditLog.objects.new(
+                action=AuditLog.PAGEVIEW,
+                user=request.user,
+                request=request,
+                project=project,
+            )
+
+    def _is_audit_enabled(self, project):
+        """
+        Check if the project has the audit feature enabled to track individual page views.
+
+        This feature is different from page views analytics,
+        as it records every page view individually with more metadata like the user, IP, etc.
+        """
+        return True
```

Comment:
Probably default this to `False` I think. No reason to have it on unless we really want it to be.

Reply:
yeah, I have it as true for testing for now.